ssl.conf

```diff
@@ -9,8 +9,9 @@ ssl_session_cache shared:SSL:50m;
 ssl_session_tickets off;
 
 # modern configuration. tweak to your needs.
-ssl_protocols TLSv1.2;
+ssl_protocols TLSv1.3 TLSv1.2;
 ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+ssl_ecdh_curve X25519:secp521r1:secp384r1;
```

Comment:
Any argument why one should limit oneself to these specific curves?
https://ssl-config.mozilla.org/ doesn't mention them

I try to keep the config non-opinionated as tls-cuves have little to do with Matomo itself but are more the decision of the server admin.

Reply:
https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility